### PR TITLE
Update the aus map cta styles

### PR DESCRIPTION
**Pull request by TomPretty**
Update the styles of the map ctas. I've updated the styles of the secondary cta to now mirror the primary one. We don't actually have an updated design for the secondary cta, but I ran it by Paula and she was happy to make them match (it's only on mobile and requires you to hit the 'read more' cta, so it will get very few views anyway).

[**Trello Card**](https://trello.com/c/bBCDpC6T/441-%F0%9F%A6%98supporter-map-update-form-embed-colour-palette-copy-survey-link)

## Screenshots

Screenshot 2022-05-18 at 10 49 10

Screenshot 2022-05-18 at 10 49 29

### CODE REVIEW

**Comment by TomPretty** [comment]
[🦘Supporter map - update form embed colour palette, copy & survey link](https://trello.com/c/bBCDpC6T/441-%F0%9F%A6%98supporter-map-update-form-embed-colour-palette-copy-survey-link)

File: support-frontend/assets/pages/aus-moment-map/components/testimonialsContainer.tsx [hunk 1]
```diff
@@ -43,18 +48,24 @@ function TestimonialCtaPrimary() {
...
+					Do something powerful today
+				</span>
+			</h3>
+
+			<ThemeProvider theme={buttonThemeBrandAlt}>
+				<LinkButton
+					className="testimonial-cta-primary-link-button"
+					priority="primary"
+					size="small"
+					href={contributeUrl('Aus_Moment_2021_map_inline')}
```
Comment (michaelbjacobson) [inline support-frontend/assets/pages/aus-moment-map/components/testimonialsContainer.tsx:62]: Do we still want this to be `2021`?

Comment (TomPretty) [reply support-frontend/assets/pages/aus-moment-map/components/testimonialsContainer.tsx:62]: Yeah well spotted 👍 We've got a second ticket for updating all of the tracking links so was going to make that update then

Comment (michaelbjacobson) [reply support-frontend/assets/pages/aus-moment-map/components/testimonialsContainer.tsx:62]: Cool cool

**Review by michaelbjacobson — APPROVED** [review]
Looks good to me!  👍🏼

**Pull request merged**